iot/views/iot_device_views.xml

```diff
@@ -92,6 +101,17 @@
         </field>
     </record>
 
+    <record id="iot_device_filter" model="ir.ui.view">
```

Comment:
It is usually named `iot_device_view_search` according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#security-view-and-action

Reply:
OK, I'll change them. Should I request review again?